### PR TITLE
[WIP] Release draft automation

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: ci/release.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SUBPROJECTS_PROPERTIES="$DIR/../subprojects/gradle.properties"
+ROOT_PROPERTIES="$DIR/../gradle.properties"
+DRY_RUN=0
+
+for arg in "$@"
```
Comment (sergioboi) [inline ci/release.sh:10]: Comment on what is happening here (

File: ci/release.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```
Comment (eugene-krivobokov) [inline ci/release.sh:1]: How about kotlin script? 
It'd be easier to maintain.

**Comment by sergioboi** [comment]
Sorry but my bash skills don't help me to read this code. I could approve because I know that you are a good engineer

File: ci/release.sh (hunk 1 above)
Comment (dsvoronin) [reply ci/release.sh:10]: that's cli arguments parsing, some basic way to do it

File: ci/release.sh (hunk 2 above)
Comment (dsvoronin) [reply ci/release.sh:1]: i don't mind to try kts here

**Comment by dsvoronin** [comment]
> Sorry but my bash skills don't help me to read this code. I could approve because I know that you are a good engineer

best signal to rewrite it to kts :)

File: subprojects/gradle/infra-release/src/main/kotlin/com/avito/android/Release.kt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        currentBranch: String
+    ) {
+        logger.info("Creating release draft $releaseTag")
+
+        if (currentBranch != releaseTag) {
+            logger.warn("Release initiated from $currentBranch, but should run on release branch: $releaseTag")
+            return
+        }
+
+        //tag
```
Comment (eugene-krivobokov) [inline subprojects/gradle/infra-release/src/main/kotlin/com/avito/android/Release.kt:24]: Do you want to merge PR in this state?
If not, can you mark PR as WIP?

File: subprojects/gradle/infra-release/src/main/kotlin/com/avito/android/Release.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        //push
+
+        val log = git.log(beginTag = previousReleaseTag, endTag = releaseTag)
+
+        log.fold(
+            { logger.info(it) },
+            { error -> logger.critical("Can't get commit log", error) }
+        )
+
+        //gh release create
```
Comment (eugene-krivobokov) [inline subprojects/gradle/infra-release/src/main/kotlin/com/avito/android/Release.kt:34]: Will it be a draft?

**Comment by dsvoronin** [comment]
will return to this PR later

**Pull request closed without merging**